site/snippets/layouts/menu.php

```diff
@@ -6,67 +6,86 @@
 	<nav aria-label="Main menu">
 		<ul class="menu-1">
 			<li class="has-submenu">
-				<a href="/features/developers">The CMS</a>
+				<a href="<?= page('features/developers')->menuUrl() ?>">The CMS</a>
 				<ul class="menu-2">
-					<li><a href="/features/developers">For developers</a></li>
-					<li><a href="/features/designers">For designers</a></li>
-					<li><a href="/features/creators">For content creators</a></li>
-					<li><a href="/features/clients">For clients & agencies</a></li>
-					<li><hr /></li>
-					<li><a href="/love">Showcase</a></li>
-					<li><a href="/releases">Releases</a></li>
-					<li><a class="is-external" href="https://feedback.getkirby.com">Feedback</a></li>
+					<?php snippet('layouts/menu-items', [
+						'externalClass' => 'is-external',
+						'items' => [
+							'For developers' => page('features/developers'),
+							'For designers' => page('features/designers'),
+							'For content creators' => page('features/creators'),
+							'For clients & agencies' => page('features/clients'),
+							'hr',
```

Comment:
```suggestion
							'-',
```

Would be more consistent with Kirby itself (e.g. Panel menu).